README.md

```diff
@@ -141,6 +141,26 @@ and pierre.casteran@labri.fr.
   patches from Coq developers when they introduce a breaking change (this is
   particularly recommended for plugins).
 
+- **What license to choose for a coq-community project?**
+
+  The only strict requirement is to use an open source license.
+  However, if you create a new project or propose to transfer a
+  project of which you are the sole copyright owner, we strongly
+  encourage you to (re)license your project under one of the following
+  two licenses:
+
+  - [MIT license](https://choosealicense.com/licenses/mit/): the most
+    permissive and most popular open source license.  This is the best
+    choice if you want to maximize reusability of your project.
+
+  - [MPL-2.0 license](https://choosealicense.com/licenses/mpl-2.0/): a
+    weak copyleft license.  You can use this license if you want to
+    restrict the license under which modified versions of your project
+    may be distributed.  It doesn't limit how larger works may depend
+    on your project.  This license should be preferred over the
+    (historically more prevalent) LGPL license because it is
```

Comment:
Can we link to the most recent LGPL here maybe? Not everybody knows what it is. And it might be called "the LGPL family of licenses" (there was the library GPL, then LGPL-2.1, then LGPL-3.0).

Reply:
The problem with linking to the most recent LGPL version is that the actually most prevalent one in the Coq & OCaml ecosystem is version 2.1 (version 2.0 is virtually non-existent and version 3.0 is used by some). Should I put a link to the LGPL version 2.1 (https://choosealicense.com/licenses/lgpl-2.1/). The issue with adding a link is that it does not seem very compatible with the idea of changing the text to talk about the "LGPL family of licenses".

Reply:
Yes, it makes sense to only link to LGPL-2.1 in this case, so you could write:
```
This license should be preferred over the (historically more prevalent)
[LGPL-2.1 license](https://choosealicense.com/licenses/lgpl-2.1/) because it is [...]
```